Add system requirements to README. (#155)
algorand/conduit: README.md

* Send blockchain data to another streaming data platform for additional processing (e.g. RabbitMQ, Kafka, ZeroMQ).
* Build an NFT catalog based on different standards.

# System Requirements

For a simple deployment the following configuration works well:
* Network: Conduit colocated with Algod follower.
* Conduit: 4 CPU and 8 GB of ram.

> * Algod: 4 CPU and 8 GB of ram.

Reviewer: Perhaps `Conduit + Algod: 4 CPU and 8 GB of RAM`? Otherwise it implies needing to dedicate that much to each process.

> * Conduit + Algod colocated: 4 CPU and 8 GB of ram.
>   * You may want to set [GOMEMLIMIT](https://pkg.go.dev/runtime@master#hdr-Environment_Variables) for Algod (and even Conduit) based on your deployment configuration. 

Author: ```suggestion
* Conduit + Algod colocated: 4 CPU and 8 GB of ram.
  * Deployments allocating less ram might work in conjunction with [GOMEMLIMIT](https://pkg.go.dev/runtime@master#hdr-Environment_Variables) for Algod (and even Conduit). This configuration is not tested, so use with caution and monitor closely. 
```

> * Storage: 20 GiB, 3000 IOPS minimum.

Reviewer 2: ```suggestion
* Storage: 40 GiB, 3000 IOPS minimum.
```

Reviewer: I'd remove this storage line altogether actually. It's inherent in the Conduit + Algod section above.

> 

Reviewer 2: ```suggestion
```

* Send blockchain data to another streaming data platform for additional processing (e.g. RabbitMQ, Kafka, ZeroMQ).
* Build an NFT catalog based on different standards.

# System Requirements

For a simple deployment the following configuration works well:
* Network: Conduit colocated with Algod follower.
* Conduit + Algod: 4 CPU and 8 GB of ram.
  * Storage: algod follower node, 40 GiB, 3000 IOPS minimum.
  * Deployments allocating less ram might work in conjunction with [GOMEMLIMIT](https://pkg.go.dev/runtime@master#hdr-Environment_Variables) for Algod (and even Conduit). This configuration is not tested, so use with caution and monitor closely.

> * Storage: 20 GiB, 3000 IOPS minimum.

Reviewer: ```suggestion
```
